packages/vite/rollup.config.js

```diff
@@ -275,6 +275,20 @@ function licensePlugin() {
       const coreLicense = fs.readFileSync(
         path.resolve(__dirname, '../../LICENSE')
       )
+      function sortLiscense(licenses) {
```

Comment:
typo in the function name:
```suggestion
      function sortLicense(licenses) {
```

Reply:
Thanks!